### PR TITLE
Add lib-rca sub module

### PR DESCRIPTION
**Pull request by RockyMandayam2**
Add lib-rca sub module

### CODE REVIEW

**Review by wuweicaifb — COMMENTED** [review]
Please don't include `.DS_Store` in the commit.

File: lib-rca/src/main/java/com/facebook/openwifi/librca/RootCauseAnalyzer.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+ */
+
+package com.facebook.openwifi.librca;
+
+public class RootCauseAnalyzer {
+
+	/** Main method. Placeholder code. */
+	public static void main(String[] args) {
+		System.out.println("Placeholder.");
+	}
```
Comment (elludraon) [inline lib-rca/src/main/java/com/facebook/openwifi/librca/RootCauseAnalyzer.java:16]: FYI, you don't need a main method

File: lib-rca/README.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# Root Cause Analysis (RCA) Java Library
+A Java library which analyzes statistics and provides root cause analysis (RCA) for clients.
```
Comment (elludraon) [inline lib-rca/README.md:2]: Can you add another line like "This is a work in progress." to be clear?

File: .gitignore [hunk 3]
```diff
@@ -17,3 +17,7 @@ bin/
 
 # Miscellaneous files thzt should not be checked in
 temp/
+
+
+# .DS_Store files
```
Comment (elludraon) [inline .gitignore:22]: nit: delete the comment

File: .gitignore [hunk 4]
```diff
@@ -17,3 +17,7 @@ bin/
...
+*.DS_Store
```
Comment (elludraon) [inline .gitignore:23]: You don't need the `*` prefix I think?

**Review by elludraon — APPROVED** [review]

File: lib-rca/README.md (hunk 2 above)
Comment (RockyMandayam2) [reply lib-rca/README.md:2]: Yes

File: .gitignore (hunk 4 above)
Comment (RockyMandayam2) [reply .gitignore:23]: I'll remove it

File: lib-rca/src/main/java/com/facebook/openwifi/librca/RootCauseAnalyzer.java (hunk 1 above)
Comment (RockyMandayam2) [reply lib-rca/src/main/java/com/facebook/openwifi/librca/RootCauseAnalyzer.java:16]: I'll remove it

**Review by wuweicaifb — APPROVED** [review]

**Review by zhiqiand — APPROVED** [review]

**Pull request merged**